Frameworks/CoreFoundation/NumberDate.subproj/CFTimeZone.c

```diff
@@ -53,26 +55,12 @@
 #include <Foundation/NSTimeZone.h>
 #include <Foundation/NSDate.h>
 
-struct tzhead {
```

Comment:
We (ms-jihua and I) had discussed this before.  It is an option, but it doesn't buy us much given that this is one file that is an exception from what we have seen.  Perhaps there are others that will fall into this bucket specifically stream, runloop, etc.  

---
In reply to: [72732284](https://github.com/Microsoft/WinObjC/pull/701#discussion_r72732284) [](ancestors = 72732284)